src/sentry/ingest/transaction_clusterer/tasks.py

```diff
@@ -14,6 +15,8 @@
 from .meta import track_clusterer_run
 from .tree import TreeClusterer
 
+logger_transactions = logging.getLogger("sentry.ingest.transaction_clusterer.tasks")
```

Comment:
This is how it's done in most modules:

```suggestion
logger = logging.getLogger(__name__)
```

Reply:
I wanted to avoid that approach as it will have the same logger name for the transaction and span clusterer, which may result in a non-desired grouping.